src/draw/vg_lite/lv_draw_vg_lite.c

```diff
@@ -112,36 +117,49 @@ static void draw_execute(lv_draw_vg_lite_unit_t * u)
     vg_lite_identity(&u->global_matrix);
     vg_lite_translate(-layer->buf_area.x1, -layer->buf_area.y1, &u->global_matrix);
 
+    LV_LOG_USER("draw task: %d", t->type);
```

Comment:
Please remove this line, or change to TRACE, because it looks like a "debugging" / trace log event.